src/stytch-provider.ts

```diff
@@ -72,10 +210,15 @@ function StytchProvider(this: any, options: StytchProviderOptions) {
     let project_id = res.keymap.project_id.value
     let secret = res.keymap.secret.value
 
+    const agent = new https.Agent({
```

Comment:
is this recommended? why is it needed
a comment is a good idea in this case to explain unexpected code
(one expects the SDK to "just work")

Reply:
yes - recommended.

Reply:
thus please add a comment